pages/common/wrk.md

```diff
@@ -0,0 +1,16 @@
+# wrk
+
+> HTTP benchmarking tool.
+> Homepage: <https://github.com/wg/wrk>.
+
+- Run a benchmark for `30` seconds, using `12` threads, and keeping `400` HTTP connections open:
+
+`wrk -t{{12}} -c{{400}} -d{{30s}} http://127.0.0.1:8080/index.html`
+
+- Run a benchmark with header:
```

Comment:
Perhaps this will read better?
```suggestion
- Run a benchmark with a custom header:
```

Also, don't `2`, `5`, and `5s` need to be enclosed in `{{` and `}}`?